packages/craftcms-webpack/index.js

```diff
@@ -84,7 +84,7 @@ const getConfig = ({
     );
   }
 
-  const configName = path.basename(context);
```

Comment:
In case a name was passed with the config, we should use that. That will happen anyway, because we merge `config` at the end, but this seems more clear in case we do anything else with `configName` and expect it to be right.

Reply:
Works for me!